etl/sql/stem/insert_cis_into_stem.py

```diff
@@ -100,6 +101,8 @@ def create_simple_stem_insert(
         concept_lookup_stem_cte.c.timezone,
     )
 
+    cl1 = aliased(ConceptLookup)
```

Comment:
why alias and not just use ConceptLookup directly in the query? Do you use ConceptLookup elsewhere in the query? If you do, I'd use two aliases to make it more explicit and, perhaps, give them informative names such as ConceptLookupRoute

Reply:
Agree and yes I am using elsewhere . I have replaced with the alias ConceptLookupRoute and ConceptLookupValue